hastexo/hastexo.py

```diff
@@ -212,19 +212,16 @@ def parse_attributes(tag, node, block):
             name = node.attrib["name"]
             if not name:
                 raise KeyError("name")
-            capacity = node.attrib.get("capacity")
-            if capacity in (None, "None", ""):
-                capacity = -1
```

Comment:
Hold on, I'm not sure if removing this is correct. The logic here is meant to be:

- if `capacity` is set to a negative value, then it's unlimited
- if it's set to 0, then it's actually 0 (i.e. don't run anything here)
- if it's set to a positive value, then that's the actual limit.

Are you sure your changes preserve that?

Reply:
This change drops the `capacity` attribute when it is for some reason set to None, "None" or "". If it is set to a negative value or 0 these values will be exported and imported as they are. However, I am not sure if the absence of the `capacity` attribute altogether will make spinning up labs to fail. If so then we might not be able to apply the same logic for `capacity` as we do for `template`.

Reply:
OK, it would be good to have a regression test for this. Can you please update `test_tasks.py`, adding test cases for an undefined `capacity`?

Reply:
Ok, we cannot have `capacity` undefined, labs will fail with that. I reverted this part and added additional tests for it.